docs/guides/manage-translations.rst

```diff
@@ -0,0 +1,229 @@
+
+.. _transifex-client: https://docs.transifex.com/client/introduction
+
+.. code-block:: console
+
+   $ pip install transifex-client
+
+After installing it, you need to configure your account.
+For this, you need to create an API Token for your user to access this service through the command line.
+This can be done under your `User's Settings`_.
+
+.. _User's Settings: https://www.transifex.com/user/settings/api/
+
+
+Now, you need to setup it to use this token:
+
+.. code-block:: console
+
+   $ tx init --token $TOKEN --no-interactive
+
+
+The next step is to map every ``.pot`` file you have created in the previous step to a resource under Transifex.
+To achieve this, you need to run this command:
+
+.. code-block:: console
+
+   $ tx config mapping-bulk --project $TRANSIFEX_PROJECT --file-extension '.pot' --source-file-dir docs/_build/gettext --source-lang en --type PO --expression 'docs/locale/<lang>/LC_MESSAGES/{filepath}/{filename}.po' --execute
+
+This command will generate a file at ``.tx/config`` with all the information needed by the ``transifext-client`` tool to keep our translation synchronized.
+
+Finally, you need to upload these files to Transifex platform so translators can start their work.
+To do this, you can run this command:
+
+.. code-block:: console
+
+   $ tx push --source
+
+
+Now, you can go to your Transifex's project and check that there is one resource per ``.rst`` file of your documentation.
+
+After the source files are translated using Transifex, you can download the all translations for all the languages by running:
+
+.. code-block:: console
+
+   $ tx pull --all
+
+This command will leave the ``.po`` files needed for building the documentation in the target language under ``locale/<lang>/LC_MESSAGES``.
+
+.. warning::
+
+   It's important to use always the same method to translate the documentation and do not mix them.
+   Otherwise, it's very easy to end up with inconsistent translation or loosing already translated text.
+
+
+Build the documentation in target language
+------------------------------------------
+
+
+Finally, to build our documentation in Spanish(Argentina) we need to tell Sphinx builder the target language with the following command:
+
+.. code-block:: console
+
+   $ sphinx-build -b html -D language=es_AR . _build/html/es_AR
+
+.. note::
+
+   There is no need to create a new ``conf.py`` to redefine the ``language`` for the Spanish version of this documentation.
+
+After running this command, the Spanish(Argentina) version of your documentation will be under ``_build/html/es_AR``.
+
+
+Summary
+-------
+
+Update sources to be translated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once we have done changes in our documentation, we want to make these additions/modifications available for translators so they can update it:
+
+#. Create the ``.pot`` files:
+
+   .. code-block:: console
+
+      $ sphinx-build -b gettext . _build/gettext
+
+
+.. For the manual workflow, we need to run this command
```

Comment:
I'm far (like, quite) from an rst expert, and I don't know enough of the syntax to understand the `#.` but the fact this one is the only one to begin differently seems noticeable (I bet I'm gonna learn something today :D )

Reply:
He he :)

The `#.` is to define a enumerated (automatically) list: http://docutils.sourceforge.net/docs/user/rst/quickref.html#enumerated-lists

When the line starts with `.. ` it's just a comment that won't appear in the renderized (html, pdf, etc) version: http://docutils.sourceforge.net/docs/user/rst/quickref.html#comments

Reply:
But then I don't understand wyl this line in particular should be a comment ?

(Or was your "he he" a mark that it shouldn't ?)

Reply:
> But then I don't understand wyl this line in particular should be a comment ?

I noted as a comment because I wanted to mention it but I thought that it breaks the summary that I was describing, which it was all based on Transifex since it's the one that _we_ use under RTD.

> (Or was your "he he" a mark that it shouldn't ?)

Nope, it didn't mean that. Just made me fun that there was a comment in the middle of nowhere without any kind of explanation :)

I'm not sure yet how to include it. I mean, maybe this section needs some more work to make the mention of these command in a proper way.